### ISSUE TITLE
Export of Project data to JSON

### ISSUE THREAD

**Issue by mcjaeger** [eclipse/sw360#1087]
t.b.c.

**Comment by KoukiHama** [comment]
Hi. @mcjaeger 
Are there any update of this issue ?
Our user are also interested in this export function.

**Comment by mcjaeger** [comment]
Yes, no, not an update of the issue, @KoukiHama I am sorry for my late answer. Technically, it could be a REST call, but the challenge lies in the attachments and some UI-based download/upload of some all-contained package. Maybe it could be a similar technique as the liceneses archive. However, no requirements or simiar so far there. Just the idea that all information of a project can be downloaded and after some time or at another instance uploaded again. I guess.

**Comment by KoukiHama** [comment]
@mcjaeger  Thank you for reply.
I understand this issue in detail. 
> the challenge lies in the attachments and some UI-based download/upload of some all-contained package.

Nice idea.  If realize this one on GUI, it will be very useful. 

> Technically, it could be a REST call, 

And, I made python script which import/export almost all these. 
- attachments
- lineked releases
- attachments of the linked releases according to usage information
- linked projects ... and then ... also the same thing recursively ...

If my this python script is helpful for sw360 community, I may be able to contribute it.
